src/map/core/MapView.jsx

```diff
@@ -50,7 +50,6 @@ const initMap = async () => {
       });
     });
   }
-  updateReadyValue(true);
```

Comment:
Why are we changing this?

Reply:
Can move it back if preferred - was something that took me a moment to work out and seems more logical to have the before and after events reflecting each other

Reply:
If it's only called in one place, it's probably fine to keep.